lab-jeff/README.md

```diff
@@ -0,0 +1,29 @@
+![Triforce](./triforce2.gif)
+
+#TCP Chat Server#
+
+###Summary###
+A TCP chat server/client program running in Node.js
+
+###Instructions###
+From the root directory of 'lab-jeff' run the following command in your terminal:
+
+`npm install`
+
+Then to start the server:
+
+`node ./lib/server.js`
+
+Finally to connect new clients, open a new tab for each client and run:
+
+`telnet localhost 300`
+
+You can then chat by typing your message and pressing enter.
```

Comment:
You should also include instructions for how to end a chat session, by typing 'END' in your case.